draftjs_exporter/options.py

```diff
@@ -30,37 +32,38 @@ def __ne__(self, other):
         return not self == other
 
     @staticmethod
-    def for_block(block_map, type_):
-        if type_ not in block_map:
-            raise ConfigException('Block "%s" does not exist in block_map' % type_)
-
-        block = block_map.get(type_)
+    def for_kind(kind_map, type_, fallback_key):
+        """
+        Create an Options object from any mapping.
+        """
+        if type_ not in kind_map:
+            if fallback_key not in kind_map:
+                raise ConfigException('"%s" is not in the config and has no fallback' % type_)
+
+            config = kind_map[fallback_key]
+        else:
+            config = kind_map[type_]
 
-        if isinstance(block, list):
-            raise ConfigException('Block "%s" uses unsupported list-style config' % type_)
-        elif isinstance(block, dict):
-            if 'element' not in block:
-                raise ConfigException('Block "%s" does not define an element' % type_)
+        if isinstance(config, list):
```

Comment:
Because Python encourages duck typing, this test and the following ones could fail (e.g. `isinstance(config, list)` will fail if the object is a list like but not a list, or is a `tuple`, etc), which will fallback to the `else` clause and return a mis-configured `Options` object.

Assuming that the current implementation expects an actual class (the element) when it reaches the `else` clause, I think we can flip things around like this:

```python
from collections import Mapping
from inspect import isclass

if isclass(config):
    opts = Options(type_, config)

elif isinstance(config, Mapping):
    if 'element' not in config:
        raise ConfigException('"%s" does not define an element' % type_)
    opts = Options(type_, **config)

else:
    raise ConfigException('"%s" is improperly configured, expected element or dict' % type_)
```